code/modules/antagonists/superheroes/buzzon/buzzon_equipment.dm

```diff
@@ -0,0 +1,294 @@
+//Buzzon, controls all the robots and honey.
+
+/obj/item/clothing/gloves/tackler/combat/insulated/buzzon
+	name = "BuzzOn gloves"
+	desc = "Legendary gloves on the greatest bee superhero - BuzzOn. You can see small rocket engines installed on them."
+	icon_state = "buzz"
+	worn_icon_state = "buzz"
+	inhand_icon_state = "ygloves"
+
+	armor = list(MELEE = 60, BULLET = 60, LASER = 50, ENERGY = 60, BOMB = 55, BIO = 100, RAD = 70, FIRE = 100, ACID = 100, WOUND = 25)
+
+	tackle_stam_cost = 5 //Very light and fast tackles. However, if you fly into a wall or an enemy, well, you're fucked.
+	base_knockdown = 0.2 SECONDS
+	tackle_range = 5
+	tackle_speed = 2
+	min_distance = 2
+	skill_mod = -5
+
+/obj/item/clothing/gloves/tackler/combat/insulated/buzzon/cryo //Cosmetics for "Operation: Cryostung" kit
+	icon_state = "buzz_cryo"
+	worn_icon_state = "buzz_cryo"
+	inhand_icon_state = "bluegloves"
+
+/obj/item/clothing/shoes/sneakers/buzzon
+	name = "BuzzOn shoes"
+	desc = "Shoes of a bee that became human, rumors say that they buzz when you walk in them."
+	greyscale_colors = "#000000#f6c61a"
+
+	armor = list(MELEE = 60, BULLET = 60, LASER = 50, ENERGY = 60, BOMB = 55, BIO = 100, RAD = 70, FIRE = 100, ACID = 100, WOUND = 25)
+
+/obj/item/clothing/shoes/sneakers/buzzon/cryo
+	icon_state = "buzz_cryo"
+	worn_icon_state = "buzz_cryo"
+
+	greyscale_colors = null
+	dying_key = null
+	greyscale_colors = null
+	greyscale_config = null
+	greyscale_config_worn = null
+
+/obj/item/clothing/suit/hooded/bee_costume/buzzon
+	name = "cybernetic bee suit"
+	desc = "A complex cybernetic suit with black and yellow stripes. All hail the Queen!"
+	icon_state = "bee_full"
+
+	armor = list(MELEE = 60, BULLET = 60, LASER = 50, ENERGY = 60, BOMB = 55, BIO = 100, RAD = 70, FIRE = 100, ACID = 100, WOUND = 25)
+	body_parts_covered = CHEST|GROIN|ARMS|LEGS
+	clothing_flags = THICKMATERIAL
+
+	hoodtype = /obj/item/clothing/head/hooded/bee_hood/full
+	actions_types = list(/datum/action/item_action/toggle_hood, /datum/action/item_action/recall_beesword)
+
+	var/obj/item/melee/beesword/buzzon/linked_sword
+	var/sword_cooldown = 0
+
+/obj/item/clothing/suit/hooded/bee_costume/buzzon/cryo
+	icon_state = "bee_full_cryo"
+	hoodtype = /obj/item/clothing/head/hooded/bee_hood/full/cryo
+
+	cold_protection = CHEST|GROIN|ARMS|LEGS
+	min_cold_protection_temperature = FIRE_SUIT_MIN_TEMP_PROTECT
```

Comment:
why would I not just always choose the cryo set over the normal one, regardless of environment?

perhaps you should just give the normal suit this cold protection as well, so that choosing between them is purely an aesthetic choice

Reply:
Cryo kit is based around poison. Your stinger has much lower damage but uses frost oil instead of normal toxins